libecole/src/observation/strongbranchingscores.cpp

```diff
@@ -0,0 +1,102 @@
+#include <cmath>
+#include <cstddef>
+
+#include "ecole/observation/strongbranchingscores.hpp"
+#include "ecole/scip/type.hpp"
+
+#include <scip/scip.h>
+#include <scip/scipdefplugins.h>
+#include <scip/struct_branch.h>
+#include <scip/struct_lp.h>
+#include <scip/struct_scip.h>
+#include <scip/struct_var.h>
+#include <scip/utils.hpp>
+
+namespace ecole {
+namespace observation {
+
+static auto extract_strong_branching_scores(scip::Model const& model) {
```

Comment:
What is the point of having this separate method, which is called only once, and only here ?

Reply:
I was following the implementation of NodeBipartite to keep the style similar, but you are right it could easily be done in one method.  I don't really have a preference here so I am fine with moving it all within one method or keeping it more consistent with NodeBipartite.